src/main/webapp/css/welcome.css

```
@@ -0,0 +1,301 @@
+:root {
+    --text-01: #000000;
+    --light-01: #F9F9F9;
+    --light-02: #FFFFFF;
+    --brand-01: #F9F9F9;
+    --brand-02: #f7cd40;
+    --hover-timing: all 0.2s ease;
+    -webkit-font-smoothing: antialiased;
+    --pg-margin: 80px;
+    font-smoothing: antialiased;
```

```suggestion
    font-smooth: always;
```
Honestly, I can't see the difference with anti-aliasing turned on but I think this is how you're supposed to do it?
https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth